src/contacts/view/ContactViewerActions.ts

```diff
@@ -8,9 +8,10 @@ import { Keys } from "../../api/common/TutanotaConstants.js"
 export interface ContactViewToolbarAttrs {
 	contacts: Contact[]
 	onEdit: (contact: Contact) => unknown
-	onDelete: (contacts: Contact[]) => unknown
+	onDelete: (contacts: Contact[], onConfirm: () => void) => unknown
```

Comment:
it seems a bit odd to me, this PR adds both `onConfirm` and `selectNone` and `selectNone`  is  only used as `onConfirm`. Couldn't we just do it outside? Toolbar doesn't have an action for `selectNone` and it doesn't need to know about selection I think